### ISSUE TITLE
[BUG] Discord permissions in the "(allow|deny) permission" effects can't be 2 words

### ISSUE THREAD

**Issue by vitusverden** [SkyCraft78/DiSky#6]
**Name:** `Fix discord perms can't be 2 words`

**In-dept description of the new features:
```
Discord permissions can't be 2 words (in allow/deny permission effects)

See https://discord.com/channels/818182471140114432/818182471677247527/829032225932312617, https://discord.com/channels/818182471140114432/818182471677247527/829032251752579154, https://discord.com/channels/818182471140114432/818182471677247527/829027363450388562 and https://discord.com/channels/818182471140114432/818182471677247527/829025746190204949
```

**Comment by vitusverden** [comment]
Another example: https://discord.com/channels/818182471140114432/818182471887749125/829327593500967002

**Comment by SkyCraft78** [comment]
Fixed in 256928d
